docs/dev.rst

```diff
@@ -562,11 +622,15 @@ Now, you can send these in with ``tecken-loadtest`` like this:
    $ python upload-symbol-zips.py http://localhost:8000 -t 160 --download-url=http://localhost:9090/symbols.zip
 
 This way you'll have 3 terminals. 2 bash terminals inside the container
-and one outside in the ``tecke-loadtests`` directory on your host.
+and one outside in the ``tecken-loadtests`` directory on your host.
 
 
 Debugging a "broken" Redis
```

Comment:
I have no idea if this section is valid anymore. It pre-dates me and I've never used it, so maybe we should just delete it.